assets/src/edit-story/components/canvas/editLayer.js

```diff
@@ -43,13 +49,19 @@ const EditPageArea = withOverlay(styled(PageArea).attrs({
 `);
 
 function EditLayer({}) {
+  const ref = useRef(null);
   const {
     state: { currentPage },
   } = useStory();
   const {
     state: { editingElement: editingElementId },
+    actions: { clearEditing },
   } = useCanvas();
 
+  useKeyDownEffect(ref, { key: 'esc', input: true }, () => {
```

Comment:
Isn't this the same?
```suggestion
  useKeyDownEffect(ref, { key: 'esc', input: true }, clearEditing );
```

Reply:
Yes and no. There's some debate on [another thread](https://github.com/google/web-stories-wp/pull/320#discussion_r379643702) about this. But the callback has a signature with `Event` object. And `clearEditing` has its own signature (in this case, no arguments). It's a bit fragile if we decide to add args to `clearEditing`. E.g. `clearEditing(specialMode: boolean, ...etc)`.

Reply:
But let's continue this discussion on the other thread to dedup different conversations.